gnd/src/main/java/com/google/android/gnd/system/auth/GoogleAuthenticationManager.java

```diff
@@ -104,6 +108,7 @@ public void signIn() {
 
   public void signOut() {
     firebaseAuth.signOut();
+    localValueStore.clearTermsStatus();
     signInState.onNext(new SignInState(State.SIGNED_OUT));
```

Comment:
please consider unit test coverage for this as well.

Reply:
updated in next commit.

Reply:
added test coverage for this as well